Library/Tests/ViewModels/BackingViewModelTests.swift

```diff
@@ -138,25 +145,35 @@ internal final class BackingViewModelTests: TestCase {
 
       self.vm.inputs.viewDidLoad()
 
+      self.loadingOverlayIsHidden.assertValues([false])
+      self.backerSequence.assertDidNotEmitValue()
+
+      self.scheduler.advance()
+
+      self.loadingOverlayIsHidden.assertValues([false, true])
       self.backerSequenceAccessibilityLabel.assertValues([Strings.backer_modal_backer_number(backer_number:
         "5")],
                                        "Backer label emits backer sequence")
-
     }
   }
 
   func testBackerPledgeAmountAndDate() {
     withEnvironment(apiService: MockService(fetchBackingResponse: .template
       |> Backing.lens.amount .~ 35
       |> Backing.lens.pledgedAt .~ 1468527587.32843 )) {
-        self.vm.inputs.configureWith(project: .template |> Project.lens.country .~ .US, backer: nil)
 
-        self.backerPledgeAmountAndDate.assertValues([])
+        self.vm.inputs.configureWith(project: .template |> Project.lens.country .~ .US, backer: nil)
 
         self.vm.inputs.viewDidLoad()
 
+        self.loadingOverlayIsHidden.assertValues([false])
+        self.backerPledgeAmountAndDate.assertDidNotEmitValue()
```

Comment:
we don't need to test the loading outputs in each of our test cases, but instead let's create a separate test case for it.